doc/developers/develop.rst

```diff
@@ -35,11 +35,11 @@ multiple interfaces):
 
     The base object, implements a ``fit`` method to learn from data, either::
 
-      estimator = estimator.fit(data, targets)
+      estimation = estimator.fit(data, targets)
```

Comment:
I don't think this is correct. `fit` returns the estimator itself, by convention. I'd be happy to remove the `estimator =` since it is unnecessary.

Reply:
You're absolutely right! It seems I misread that section. I agree that your suggestion to remove `estimator=` makes more sense.